pkg/kdk/update.go

```diff
@@ -41,6 +42,11 @@ func WarnIfUpdateAvailable(cfg *KdkEnvConfig) {
 		return
 	}
 
+	// Add a sudo hint for unix-based OS's
+	sudo := ""
+	if runtime.GOOS == "linux" || runtime.GOOS == "darwin" {
+		sudo = "sudo "
```

Comment:
trailing space intentional?

Reply:
Yes, space is intentional.  Added comment